large amount of additions to ripped

ughmaires opened the pull request:
While you may or may not agree with the layout, there are some things that I feel should be implemented in the guide

kekkodance commented:
you can put linuxcracksupport back in the games section

kekkodance commented:
the Operating Systems folder seems a bit unnecessary to me, don't know about @Helmasko

ughmaires commented:
> the Operating Systems folder seems a bit unnecessary to me, don't know about @Helmasko

just thought it looked better as a subcategory, would be unorganised as Linux, Games, Windows & Software

Helmasko commented:
I'm gonna need a full run-down of every single change that has happened here, in full detail with reasoning.

ughmaires commented:
> I'm gonna need a full run-down of every single change that has happened here, in full detail with reasoning. And make sure to make this a habit, Scouts previous PRs are heavily lacking in that regard.

***Operating Systems & PC category***

- Added a specific section for each operating system, meaning that it is easier to navigate to software (and debloating on windows) for a specific operating system, and also removing the cramming of random things into Misc.md. Also added NVCleanstall and Radeon driver slimmer in order to remove telemetry and improve driver performance, This also makes it easier for Linux users to see all recommended programs and guides for gaming.

![image](https://github.com/rippedpiracy/docs/assets/79582973/ed5bc783-9aa6-4699-9835-89cbb253ae81)
![image](https://github.com/rippedpiracy/docs/assets/79582973/caadeaad-2e40-4014-9ac9-12839fc9c703)

- Reorganised the Windows debloating category, recommending that users use AME Wizard and playbooks instead of time consuming methods such as TronScript. Also added Chris Titus Tech debloat, mentioning that all of its effects are reversible and recommending it to any users who do not want to reinstall windows. This ensures that users get the best debloat options for their computers.

![image](https://github.com/rippedpiracy/docs/assets/79582973/550c0c2d-d142-4c12-9a3d-3c714438ab44)

***Utilities category***

- Made a seperate but WIP informational category, which allows users to browse privacy and piracy guides, allowing them to grasp a better understanding of the software that they are downloading. In future this can include things such as the homebrew guide in the Homebrew-WIP category.

![image](https://github.com/rippedpiracy/docs/assets/79582973/ff776f6d-4491-4d51-a049-efa0afd2ffcf)
![image](https://github.com/rippedpiracy/docs/assets/79582973/eaaacf74-09ea-4deb-9a0e-55870df55333)


- Added a web browsing category, allowing for users to quickly and easily see any plugins/extensions that are recommended without having to scroll through the site trying to eventually find it in misc.
 Removed some obsolete extensions in Web Browsing, which are included in uBlock Origin 

![image](https://github.com/rippedpiracy/docs/assets/79582973/c52f95f7-6b6a-4a62-820c-7d9e1743b3e8)

Helmasko commented:
Thanks for the detailed update. I have a few comments. Having a separate Gaming category and having gaming as a subsection in linux will cause redundancy, especially when some sites like torrminatorr and some cracking tools can be used on both OSs. I like the idea of making web-browsing a category, can you tell me which stuff you removed from there as well, and anywhere else that you didnt mention before?

I'll try to run this on my system and might come back with more feedback.

Scouts2048 commented:
> can you tell me which stuff you removed from there as well, and anywhere else that you didnt mention before?

NoScript was the only extension removed as it is redundant with UBlock. There are also filter lists that make ClearURLs unnecessary, and [Total Cookie Protection](https://blog.mozilla.org/security/2021/02/23/total-cookie-protection/) makes Privacy Badger & Disconnect redundant. However, Total Cookie Protection is only available for Firefox. The only other changes made were the removal of all software and informational content in misc, which were placed into their respective sections. PSNDL was removed from ROMs as it has been closed for months. Yuzu's Switch Emulator for Android was added to the Emulators section. The AnimePiracy Wiki URL was updated. The remaining changes mostly consist of additions to the Linux section.
